dotCMS/src/main/java/com/dotcms/api/tree/TreeableAPI.java

```diff
@@ -50,7 +50,8 @@ public TreeableAPI() {
      */
     public List<Treeable> loadAssetsUnderHost(Host host, User user, boolean live, boolean working, boolean archived, boolean respectFrontEndPermissions) throws DotDataException, DotSecurityException {
         List<Treeable> assets = new ArrayList<Treeable>();
-        assets.addAll(folderAPI.findFoldersByHost(host,user,respectFrontEndPermissions));
+        assets.addAll(folderAPI.findSubFolders(host,user,respectFrontEndPermissions));
+//        assets.addAll(folderAPI.findFoldersByHost(host,user,respectFrontEndPermissions));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)